packages/analytics/dashboard-renderer/src/mock-data.ts

```diff
@@ -33,3 +33,252 @@ export const exploreV3Response = {
     timestamp: '2023-10-15T05:00:00.000Z',
   }, { event: { STATUS_CODE_GROUPED: '5XX', REQUEST_COUNT: 17275256 }, timestamp: '2023-10-15T05:00:00.000Z' }],
 }
+
+export const timeSeriesExploreResponse = {
```

Comment:
issue: mock data should not go inside the source folder; this file will be bundled